source/blog/2010-08-07-dc2-experimental-associations-id-fields.md

```diff
@@ -33,10 +33,10 @@ sure that it works correctly and does not have to many problematic
 edge-cases. Therefore we need your feedback.
 
 -   Go to
-    [[https://github.com/doctrine/doctrine2/commits/DDC-117](https://github.com/doctrine/doctrine2/commits/DDC-117)](https://github.com/doctrine/doctrine2/commits/DDC-117)
+    [[https://github.com/doctrine/orm/commits/DDC-117](https://github.com/doctrine/orm/commits/DDC-117)](https://github.com/doctrine/orm/commits/DDC-117)
```

Comment:
`https://github.com/doctrine/orm/issues/1772`

Reply:
This is still a dead link where no code can be seen. Maybe there are still infos of the exact commits we need to link here.

Reply:
@SenseException I was trying to search for it, but can't find. I can provide the links of all commits that are related to https://github.com/doctrine/orm/blob/master/tests/Doctrine/Tests/ORM/Functional/Ticket/DDC117Test.php